contracts/Cash.sol

```diff
@@ -2,8 +2,14 @@ pragma solidity ^0.6.0;
 
 import '@openzeppelin/contracts/token/ERC20/ERC20Burnable.sol';
 import './owner/Operator.sol';
+import './interfaces/IFeeChecker.sol';
+import './interfaces/IFeeDistributor.sol';
 
 contract Cash is ERC20Burnable, Operator {
+    
```

Comment:
I would prefer the changes would be done in a Cashv2.sol file containing contract Cashv2.
This will enable us to use both Cash and Cashv2 in migration tests.

Reply:
Cash changes were reverted, and Cashv2.sol has been created